### PR TITLE
applied requests from #4

### PR DESCRIPTION
**Pull request by halotukozak**
Faker is ready. Please apply todos to `scheme.sql`, then I'll test the code.

https://github.com/halotukozak/PBD/pull/4#issue-2079143313

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -71,7 +72,7 @@ CREATE TABLE Webinar
     FOREIGN KEY (translator_id) REFERENCES Translator (id) ON DELETE SET NULL,
     FOREIGN KEY (teacher_id) REFERENCES Teacher (id) ON DELETE NO ACTION,
 
-    CONSTRAINT not_negative_webinar_price CHECK (price >= 0),
+    CONSTRAINT not_negative_webinar_price CHECK (price >= 0)
```
Comment (halotukozak) [inline schema.sql:75]: nit: trailling comma

File: schema.sql [hunk 2]
```diff
@@ -235,12 +238,12 @@ CREATE TABLE Internship
...
 )
 
-CREATE TABLE InternshipStudent
+CREATE TABLE StudentInternship
 (
     student_id    int           NOT NULL,
     internship_id int           NOT NULL,
     attended_days int DEFAULT 0 NOT NULL,
-    exam_result   int           NOT NULL,
+    exam_result   int
```
Comment (halotukozak) [inline schema.sql:246]: nit: trailling comma

File: schema.sql [hunk 3]
```diff
@@ -257,7 +260,7 @@ CREATE TABLE Meeting
     module_id               int,
     subject_id              int,
     url                     varchar(200),
-    date                    date        NOT NULL,
+    date                    datetime        NOT NULL,
```
Comment (halotukozak) [inline schema.sql:263]: I'm not sure that internships should have `dateTime`, @Corvette653?

Comment (halotukozak) [reply schema.sql:263]: If they should, change the name of field

File: schema.sql [hunk 4]
```diff
@@ -59,8 +59,9 @@ CREATE TABLE Translator
 CREATE TABLE Webinar
 (
     id            int                          NOT NULL IDENTITY (1, 1),
+    title         varchar(100)                 NOT NULL,
     price         int         DEFAULT 0        NOT NULL, --in Polish grosz
-    date          date                         NOT NULL,
+    date          datetime                         NOT NULL,
```
Comment (halotukozak) [inline schema.sql:64]: `date` -> `dateTime`, right?

File: triggers.sql [hunk 5]
```diff
@@ -67,3 +67,101 @@ BEGIN
...
+                     INNER JOIN Module M ON I.module_id = M.id
+                     INNER JOIN Course C ON M.course_id = C.id;
+        END
+
+    IF @translator_language <> @required_language
+        BEGIN
+            RAISERROR ('The language of the translator must be the same as the required language.', 16, 1);
+        END
+END;
+GO
```
Comment (halotukozak) [inline triggers.sql:167]: nit: missing new line

File: triggers.sql [hunk 6]
```diff
@@ -67,3 +67,101 @@ BEGIN
...
+    AS
+BEGIN
+    DECLARE @student_id int;
+    SELECT @student_id = (SELECT student_id FROM inserted);
+    IF EXISTS (SELECT 1
+               FROM Basket
+               WHERE student_id = @student_id
+                 AND state = 'open')
+        BEGIN
+            RAISERROR ('Student can only have one open basket at a time', 16, 1);
```
Comment (halotukozak) [inline triggers.sql:109]: nit: missing article

File: triggers.sql [hunk 7]
```diff
@@ -67,3 +67,101 @@ BEGIN
...
+    --Use here function to get the value of the parameter
+    SELECT @internship_required_attendance = (SELECT TOP 1 value
+                                              FROM Parameter
+                                              WHERE name LIKE @internship_required_attendance
+                                                AND date <= CURRENT_TIMESTAMP
+                                              ORDER BY date DESC)
+
+    IF @attended_days < @internship_required_attendance
+        BEGIN
+            RAISERROR ('Student must attend the required minimum days of internship', 16, 1);
```
Comment (halotukozak) [inline triggers.sql:90]: nit: missing article

File: schema.sql (hunk 3 above)
Comment (Corvette653) [reply schema.sql:263]: I think internships should have only a date, they last two weeks so it makes no point to hold hour

**Comment by halotukozak** [comment]
TODO(@tomaszkurcon) validate triggers and apply new schema changes

File: schema.sql (hunk 3 above)
Comment (tomaszkurcon) [reply schema.sql]: It is Meeting table, not internships.

**Comment by Corvette653** [comment]
Hope it works

**Pull request merged**